python/pickle/PyROOT_readtests.py

```diff
@@ -134,7 +134,10 @@ def test5ReadCustomTypes( self ):
 
    def test6ReadCustomTypes( self ):
       """[ROOT-10810] Test reading a RooDataSet with weights"""
-      ROOT.gEnv.SetValue("RooFit.Banner", 0)
```

Comment:
Why is this removed?

Reply:
The banner got completely removed: https://github.com/root-project/root/pull/9965

Reply:
Deserves a separate commit, then? :)

Reply:
Alright, I'll split them up :+1: